src/pyqasm/modules/qasm3.py

```diff
@@ -48,3 +49,7 @@ def accept(self, visitor):
         final_stmt_list = visitor.finalize(unrolled_stmt_list)
 
         self._unrolled_ast.statements = final_stmt_list
+
+    def draw(self, **kwargs):
```

Comment:
Can you explicitly list out the available kwargs here and notate them in the docstring? With just kwargs the function usage is cryptic on docs page and forces people to dig into source code. Need to fix this for other methods as well e.g. https://github.com/qBraid/pyqasm/issues/89